.travis.yml

```diff
@@ -14,5 +14,10 @@ matrix:
   fast_finish: true
   allow_failures:
     - php: nightly
+    - php: 5.3
 
-sudo: false
\ No newline at end of file
+before_script:
+  - composer install --prefer-dist --no-interaction --no-progress
```

Comment:
Consider adding `travis_retry` prior to this command in-case of a temporary network issue

(i.e. to read: `travis_retry composer install --prefer-dist --no-interaction --no-progress`)